examples/positioning/main/positioning.cpp

```diff
@@ -360,12 +360,37 @@ bool updateGNSSAssistance()
   }
 
   if(updateEphemeris) {
-    if(!modem.updateGNSSAssistance(
-      WALTER_MODEM_GNSS_ASSISTANCE_TYPE_REALTIME_EPHEMERIS))
-    {
-      ESP_LOGI("positioning", "Could not update real-time ephemeris data");
-      return false;
-    }
+      if(!modem.updateGNSSAssistance(
+        WALTER_MODEM_GNSS_ASSISTANCE_TYPE_REALTIME_EPHEMERIS))
+      {
+          ESP_LOGI("positioning", "Could not update real-time ephemeris data");
+          return false;
+      }
+
+      // Poll until the ephemeris data becomes available
+      const int maxWaitMs = 30000; // maximum wait time of 30 seconds
```

Comment:
```suggestion
      /* Poll until the ephemeris data becomes available */
      const int maxWaitMs = 30000; /* maximum wait time of 30 seconds */
```

Inline comments should ben encapsulated as such:

```cpp
/* Comment here. */
```

Configuration parameters should be placed up top with the others.